data/json/terrain.json

```diff
@@ -3278,6 +3278,51 @@
                 { "item": "splinter", "count": [10, 25] }
             ]
         }
+    },{
+        "type" : "terrain",
+        "id" : "t_tree_coffee",
+        "name": "coffee tree",
+        "description": "This is 'gymnocladus dioicus', or the kent coffee tree, which produces roastable coffee pods and brewable leaves in the fall.  If you examined the branches more closely, you could probably find a few mature ones.  You could also cut it down with the right tools.",
+        "symbol": "7",
+        "color": [ "light_green", "light_green", "light_green_green", "brown" ], "//": "barren in winter, fruits in autumn",
+        "move_cost": 0,
+        "flags": ["FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT"],
+        "transforms_into": "t_tree_coffee_harvested",
+        "examine_action": "harvest_ter_nectar",
+        "harvest_by_season" : [
+            { "seasons": [ "autumn" ], "entries": [ { "drop": "coffee_pod", "base_num": [ 2, 5 ], "scaled_num": [ 0, 0.5 ] } ] }
+        ],
+        "bash": {
+            "str_min": 80, "str_max": 180,
+            "sound": "crunch!",
+            "sound_fail": "whack!",
+            "ter_set": "t_dirt",
+            "items": [
+                { "item": "stick_long", "count": [3, 10] },
+                { "item": "splinter", "count": [10, 25] }
+            ]
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_tree_coffee_harvested",
+        "name": "pear tree",
```

Comment:
There is the part that says "pear tree".

Reply:
You're on the first commit man. Here's that same line in the most recent commit: https://github.com/CleverRaven/Cataclysm-DDA/pull/24800/commits/48d119b842e7f179e8a7c39b919ea37743a5f795